scripts/schema/cleaner.py

```diff
@@ -144,6 +144,9 @@ def field_or_multi_field_datatype_defaults(field_details):
         field_details.setdefault('ignore_above', 1024)
     if field_details['type'] == 'text':
         field_details.setdefault('norms', False)
+    # wildcard needs the index param stripped
+    if field_details['type'] == 'wildcard':
+        field_details.pop('index', None)
```

Comment:
Yeah I think this is safe and the right approach for now.

But the more I think about this, the more this sounds wrong to have this be the behaviour of Elasticsearch. I'm sure this is simply something that was overlooked, not a hard requirement.

Let's move forward with this PR to resolve the need for 1.7, but could you also open an issue with Elasticsearch, to ask for `index` to be supported for `wildcard` (and ideally for all types)?